cores/esp8266/uart.c

```diff
@@ -196,23 +196,30 @@ void uart_stop_isr(uart_t* uart)
 }
 
 
-void uart_write_char(uart_t* uart, char c)
+void uart_do_write_char(uart_t* uart, char c)
```

Comment:
Maybe declare this one as inline.

Reply:
Thanks. I actually don't understand why inline does save flash space here.
This means one while loop and 2 calls with 2 parameters each take 16 more flash bytes than two while loops.
I was thinking gcc -Os would optimize this itself [...] Well, it would have if I had not forgotten ```static```. Then I'd prefer the static way.